app/decorators/middleware_deployment_decorator.rb

```diff
@@ -3,11 +3,11 @@ class MiddlewareDeploymentDecorator < Draper::Decorator
 
   def fonticon
     if name.end_with? '.ear'
-      'product-file-ear-o'
+      'product product-file-ear-o'
     elsif name.end_with? '.war'
-      'product-file-war-o'
+      'product product-file-war-o'
     else
-      'product-report'
+      'fa fa-file-text-o'
```

Comment:
@himdel I meant that the "product" class that comes before "product-report" class is in another file someplace.